src/components/order/StatusLabel/index.tsx

```diff
@@ -17,29 +42,7 @@ const Wrapper = styled.div<Props>`
   padding: 0.75em;
   display: inline-block;
 
-  ${({ theme, status }): string => {
-    let background, color
-
-    switch (status) {
-      case 'expired':
-        color = theme.labelTextExpired
-        background = theme.labelBgExpired
-        break
-      case 'filled':
-      case 'partially filled':
-        color = theme.labelTextFilled
-        background = theme.labelBgFilled
-        break
-      case 'open':
-        color = theme.labelTextOpen
-        background = theme.labelBgOpen
-        break
-    }
-    return `
-      background: ${background};
-      color: ${color};
-    `
-  }}
+  ${(...params): string => setStatusColors(...params)}
```

Comment:
hmm i think you\re unnecessarily passing too many arguments here, probably better to destructure as normal?

Reply:
Just wanted to reduce the logic in the CSS as Anxo suggested.
But, no problem, will update.